infinit/license.md

```diff
@@ -0,0 +1 @@
+View [license information](https://www.infinit.sh/licenses/infinit.txt) for the software contained in this image.
```

Comment:
This URL appears to be a 404. :confused:

![image](https://user-images.githubusercontent.com/161631/26978873-cb048836-4ce1-11e7-8261-e2974073d1b4.png)